lambdas/indexer/Makefile

```diff
@@ -12,7 +12,7 @@ package: check_branch check_python check_aws config
 	# Ensure a .pyc file is present for every .py file. Set literals will
 	# compile in a non-deterministic order unless PYTHONHASHSEED is set. For a
 	# full explanation see http://benno.id.au/blog/2013/01/15/python-determinism
-	PYTHONHASHSEED=0 python -m compileall -q --invalidation-mode checked-hash vendor vendor/azul
+	PYTHONHASHSEED=0 python -m compileall -f -q --invalidation-mode checked-hash vendor vendor/azul
```

Comment:
The use of `-f` should be motivated in a comment. The PR description would be a good start. It might make sense to factor this line out into a separate `compile` target so the explanatory comments don't overwhelm the rest of the recipe.

That said, I don't quite understand the current PR description wrt `timestamp` vs `hash` so even the description might need a clarification.

Reply:
I added a more thorough description.